openedx/core/djangoapps/courseware_api/toggles.py

```diff
@@ -0,0 +1,21 @@
+"""
```

Comment:
We should check in with TNL to see if this is where this flag should exist. I believe they already have one for the courseware itself so we might want this to leave alongside that and/or make sure that this doesn't interfere with any of their flags.

Reply:
What was the outcome of this conversation @xnick421 ?